src/IssueViz/Models/AnalysisIssueVisualization.cs

```diff
@@ -42,8 +43,12 @@ public AnalysisIssueVisualization(IReadOnlyList<IAnalysisIssueFlowVisualization>
         public IReadOnlyList<IAnalysisIssueFlowVisualization> Flows { get; }
         public IAnalysisIssue Issue { get; }
         public int StepNumber => 0;
-        public bool IsNavigable { get; set; }
         public IAnalysisIssueLocation Location => Issue;
+
+        public bool IsNavigable { get; set; }
+        public string FilePath { get; set; }
         public SnapshotSpan? Span { get; set; }
+
+        public event PropertyChangedEventHandler PropertyChanged;
```

Comment:
This is temporary.
I will introduce a new interface instead, in a different PR.

```
public interface IAnalysisIssueSecondaryLocationVisualization : IAnalysisIssueLocationVisualization, INotifyPropertyChanged
    {
        int StepNumber { get; }
    }
```

Reply:
As discussed offline, let's defer this until we decide if/how we are going to show the primary location in the UI.